raffle.js

```diff
@@ -1,41 +1,158 @@
 #!/usr/bin/env node
```

Comment:
If you are using `const`, you probably need to add the least required version of node to readme file?